ConcordiumWallet/Views/Stake/Baking/ComissionSettings/BakerCommissionSettingsView.swift

```diff
@@ -0,0 +1,130 @@
+                Text("When you open your baker as a pool, you earn commissions of stake delegated to your pool from another accounts")
+                    .padding(.bottom, 16)
+
+                Text("Transaction fee commission")
+                BakerCommissionSliderView(
+                    range: ranges.transactionCommissionRange,
+                    commission: $viewModel.transactionFeeCommission
+                )
+
+                Text("Baking reward commission")
+                BakerCommissionSliderView(
+                    range: ranges.bakingCommissionRange,
+                    commission: $viewModel.bakingRewardCommission
+                )
+
+                Text("Finalization reward commission")
+                BakerCommissionSliderView(
+                    range: ranges.finalizationCommissionRange,
+                    commission: $viewModel.finalizationRewardCommission
+                )
+                Spacer()
+                Button(action: viewModel.continueButtonTapped) {
+                    Text("Continue")
+                        .padding()
+                        .frame(maxWidth: .infinity)
+                        .foregroundColor(Pallette.whiteText)
+                        .background(Pallette.primary)
+                }
+
+                .background(Pallette.primary)
+                .cornerRadius(10)
+            } else {
+                Spacer()
+                LoadingIndicator()
+                Spacer()
+            }
+        }
+        .alert(
+            "Error",
+            isPresented: $viewModel.error.isNotNil(),
+            presenting: viewModel.error,
+            actions: { error in
+                Button {
+                    switch error {
+                    case .networkError:
+                        viewModel.dismissView()
+                    default: break
+                    }
+                    viewModel.error = nil
+                } label: {
+                    Text("OK")
+                }
+            },
+            message: { error in
+                Text(error.errorMessage)
+            }
+        )
+        .onAppear { viewModel.fetchData() }
+        .padding()
+    }
+}
+
+
+struct BakerCommissionSliderView: View {
+    var range: CommissionRange
+    @Binding var commission: Double
+    let formatter: NumberFormatter = .commissionFormatter
+
+    var body: some View {
+        let commissionBinding = Binding<Double>(get: {
+            self.commission
+        }, set: {
+            var commission = $0
+            // Snap to the range limits to prevent odd floating point roundoff.
+            if commission > range.max - BakerCommissionSettingsView.sliderStep {
+                commission = range.max
+            }
+            if commission < range.min + BakerCommissionSettingsView.sliderStep {
+                commission = range.min
+            }
+            self.commission = commission.rounded(.toNearestOrAwayFromZero, decimals: 5)
```

Comment:
```suggestion
            self.commission = commission.rounded(.toNearestOrAwayFromZero, decimals: 3)
```

As `commissionFormatter` only shows 3 digits, any digits following that will be just noise?